9.2/run-postgresql.sh

```diff
@@ -98,6 +109,14 @@ function initialize_database() {
 # additional custom settings and is included from $PGDATA/postgresql.conf.
 envsubst < ${POSTGRESQL_CONFIG_FILE}.template > ${POSTGRESQL_CONFIG_FILE}
 
+# Generate passwd file based on current uid
+export USER_ID=$(id -u)
+envsubst < ${HOME}/passwd.template > ${HOME}/passwd
+cat ${HOME}/passwd
+export LD_PRELOAD=libnss_wrapper.so
+export NSS_WRAPPER_PASSWD=/var/lib/pgsql/passwd
+export NSS_WRAPPER_GROUP=/etc/group
```

Comment:
why did we end up needing all this? postgres insists on having a user named postgres?